backends/xnnpack/CMakeLists.txt

```diff
@@ -113,8 +114,13 @@ if(NOT CMAKE_TOOLCHAIN_FILE MATCHES ".*(iOS|ios\.toolchain)\.cmake$")
   #
   list(TRANSFORM _xnn_executor_runner__srcs PREPEND "${EXECUTORCH_ROOT}/")
   add_executable(xnn_executor_runner ${_xnn_executor_runner__srcs})
+
+  if(EXECUTORCH_ENABLE_EVENT_TRACER)
+    list(APPEND xnn_executor_runner_libs etdump)
```

Comment:
I guess this never worked for OSS. Thanks. We should also consider adding `ENABLE_XNNPACK_PROFILING` CMake flag here.

Reply:
The flag `ENABLE_XNNPACK_PROFILING` is not used by the executor_runner to enable etdumps, so I thought it should be fine to just use the `EXECUTORCH_ENABLE_EVENT_TRACER`.

Reply:
Yeah not an issue with your PR.